tests/system/src/test/java/test/javafx/scene/InitializeJavaFXTest.java

```diff
@@ -0,0 +1,84 @@
+package test.javafx.scene;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.stage.Stage;
+import junit.framework.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+public class InitializeJavaFXTest {
+
+    public static final CountDownLatch appLatch = new CountDownLatch(1);
+
+    public static class InitializeApp extends Application {
+        @Override
+        public void start(Stage primaryStage) throws Exception {
+            appLatch.countDown();
+        }
+    }
+
+    public static void initializeApplication() throws Exception {
+        new Thread(() -> {
+            Application.launch(InitializeApp.class);
+        }).start();
+        appLatch.await();
+    }
+
+    public static void initializeStartup() throws Exception {
+        CountDownLatch latch = new CountDownLatch(1);
+        Platform.startup(() -> {
+            latch.countDown();
+        });
+        latch.await();
```

Comment:
This needs to be changed to a flavor of await with a timeout (you can assert that it doesn't timeout). Also, I don't think this needs to be its own method, since the only thing the `initialize` method does is call this.